src/api/client/AccountManagement.ts

```diff
@@ -122,6 +141,13 @@ export class AccountManagement {
    */
   public removeSecondaryAccounts: ProcedureMethod<RemoveSecondaryAccountsParams, void>;
 
+  /**
+   * Adds a list of secondary Accounts to the signing Identity
+   *
+   * @throws if the signing Account is not the primary Account of the Identity
+   */
+  public addSecondaryAccounts: ProcedureMethod<AddSecondaryAccountsParams, Identity>;
```

Comment:
The comment should mention how to get the auth signatures, and why its better than invite account, since they perform similar functions.